components/app/BusinessInfo.jsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 
 import { isLast } from '../utils';
 
-import { Box, Text, Image } from '../atoms';
+import { Box, Text, categoryImage } from '../atoms';
```

Comment:
Hi, just to point out that using camel case for React components is quite error prone. I'd suggest using `CategoryImage`.

If it's used in JSX, `<categoryImage>` will resolve to `{ type: 'categoryImage' }` -> `document.createElement('categoryImage')`, whereas `<CategoryImage>` will resolve the type to the next lower case element type: `{ type: 'img' ` from `styled.img`.

Also, it's fine to leave, but I would have the `CategoryImage` in `app/CategoryImage`, since by the name of the component, it seems more coupled to the app state, rather than a design primitive.

Reply:
Ah thanks for letting me know - I think this is why the CI tests were failing! I've changed the case and moved the file - I agree it makes more sense to have the file in app.